firstvoices/backend/search/documents/base_document.py

```diff
@@ -2,8 +2,14 @@
 
 
 class BaseDocument(Document):
+    """Basic fields used for generic hydration utils"""
+
+    document_id = Keyword()  # model id
+    document_type = Keyword()  # model class name
```

Comment:
Just wondering if this should sit in the base doc ? since we generally store diff. models in diff. indices.
If this is required for language index, shall we move this in the specific document ?

Reply:
Great question! This is required for the hydration steps in the BaseSearchView, so I left these fields in the base document. I logged a couple of tickets for using the generic hydration for other search APIs if possible as well-- if it turns out we can't make it work we can move all the "general" things right into the language search classes. Cool?

Reply:
Sounds good.
There is also the `obj["_index"]` which we can map/use directly for hydration or to be returned in the response. We can add this change later though.